model_compression_toolkit/core/pytorch/graph_substitutions/substitutions/scaled_dot_product_attention.py

```diff
@@ -68,8 +68,8 @@ def _get_transpose_k_node(self, attention_node_name: str, key_node: BaseNode) ->
         output_shape[-2], output_shape[-1] = input_shape[-1], input_shape[-2]
         transpose_node = FunctionalNode(name=f"{attention_node_name}_{key_node.name}_transpose",
                                         framework_attr={},
-                                        input_shape=input_shape,
-                                        output_shape=output_shape,
+                                        input_shape=[input_shape],
```

Comment:
is this change related to the max cut integration? if not, why is it failing all of a sudden? (similar question to the other changes in this file)